Adding recommended configuration for 500 node cluster and miscellaneous fixes

divyenpatel opened the pull request:
Here is the summary of change.
 - Updated `Prerequisites` title in `home_sidebar.yml`
 - Updated `faqs.md` and mentioned that VCP is tested with 500 node cluster so far.
 - Added recommended configuration for large cluster.

rjog commented on documentation/faqs.md:
```diff
@@ -2,17 +2,22 @@
-
 ## What is the biggest Kubernetes cluster it has been tested for?
-It has been tested on eight node cluster so far.
+It has been tested on 500 node cluster so far. Please refer to this [section](/vsphere-storage-for-kubernetes/documentation/largescaledeployment.html) for details about recommended configuration.
```
Will it make more clear with "500 node kubernetes cluster"?
In our environment, we have ESXi cluster, vSAN cluster, cabinets cluster.. too many clusters!

divyenpatel replied:
fixed.

rjog commented on documentation/faqs.md:
```diff
@@ -2,17 +2,22 @@
+## Which vSphere Datastore types are supported by vSphere Cloud Provider?
+VMFS, NFS, vSAN and VVOl datastore types are supported by vSphere Cloud Provider.
+
+## Where is the details about Storage Policy Based Management (SPBM) for dynamic provisioning of volumes?
```
Q should be:

"Can SPBM be used with dynamic volume provisioning?"

divyenpatel replied:
changed question.

rjog commented on documentation/faqs.md:
```diff
@@ -2,17 +2,22 @@
+## Where is the details about Storage Policy Based Management (SPBM) for dynamic provisioning of volumes?
+Please refer to this [section](/vsphere-storage-for-kubernetes/documentation/policy-based-mgmt.html). 
+Also refer to [Storage Policy Management inside kubernetes](https://github.com/kubernetes/examples/blob/master/staging/volumes/vsphere/README.md#storage-policy-management-inside-kubernetes) for more details.
+Please report in case you find any features are missing.
```
This should be generically told somewhere else.
May be at the bottom of this page.

divyenpatel replied:
removed this line.

rjog commented on documentation/largescaledeployment.md:
```diff
@@ -0,0 +1,77 @@
+
+Worker nodes are allocated 2 GB and 2 CPUs.
+
+**Note:** CPU/Memory allocation for nodes can be adjusted based on the workload requirement. Above is the minimum recommended allocation.
```
Above is the minimum required allocation.

divyenpatel replied:
Fixed.

rjog commented on documentation/largescaledeployment.md:
```diff
@@ -0,0 +1,77 @@
+
+**Note:** Here resource `requests` is set on system pods and not the `limit`, so that if Pod requires more CPU/Memory then available CPU/Memory can be utilized during the peak time.
+
+For rest of the Pods on the master node (Proxy, DNS and flannel) we have 40% cpu available.
```
(Proxy, DNS and flannel etc.)

divyenpatel replied:
Fixed.

divyenpatel commented:
@rohitjogvmw Addressed review comments https://github.com/vmware/vsphere-storage-for-kubernetes/pull/14/commits/f8d30a675c396de6b0b6d0ac4452811d779ef051
